src/main/kotlin/com/viral32111/discordrelay/discord/data/ApplicationCommandInteractionDataOption.kt

```diff
@@ -0,0 +1,15 @@
+package com.viral32111.discordrelay.discord.data
+
+import kotlinx.serialization.Required
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+// https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-application-command-interaction-data-option-structure
+@Serializable
+data class ApplicationCommandInteractionDataOption(
+    @Required @SerialName("name") val name: String,
+    @Required @SerialName("type") val type: Int,
+    @SerialName("value") val value: String? = null,
```

Comment:
Add a comment to explain why value field is different from spec (eg. "Not necessary to implement other value types, we only receive strings")